### DISCUSSION TITLE
CronJob: Process Manager bundle

### DISCUSSION THREAD

**Discussion by Jimerson21** [pimcore/pimcore#9515] [Q&A / Support]
Hello everyone,
I need to configure a crontab with the Process-Manager package.

Add the cronjob expression:
          * * * * *

and configure the crontab (crontab -e):

      * * * * * docker exec -it --user www-data container-name php bin/console process-manager:maintenance > /dev/null 2>&1

it never runs automatically, but if I try it manually it works. What am I doing wrong?

Thanks.

**Comment by aweichler** [comment]
Hello.
I think you need to call bin/console with absolute path in the crontab file.
For example "/var/www/bin/console".

**Reply by Jimerson21** [reply]
Thanks for your answer, I managed to solve it. I just had to remove the It from the docker command ;)
